apps/src/componentLibrary/typography/components/BodyOneText.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Typography, {VisualApproach} from './../index';
+
+interface BodyOneTextProps {
+  visualApproach?: VisualApproach;
+  className?: string;
+  style?: React.CSSProperties;
+  children: React.ReactNode;
+}
+
+const BodyOneText: React.FunctionComponent<BodyOneTextProps> = ({
+  visualApproach,
+  className,
+  style,
+  children
+}) => {
+  return (
+    <Typography
```

Comment:
Also seems like body of these are mostly the same, besides the `semanticTag` prop...maybe we could pull this into some common generator function?